src/neuroglancer/datasource/graphene/frontend.ts

```diff
@@ -949,6 +949,27 @@ class GraphConnection extends SegmentationGraphSourceConnection {
     return undefined;
   }
 
+  getMeshSource() {
```

Comment:
This should somehow ensure that it is associated with this graph --- can have more than one datasource in a UserLayer.

Reply:
Would this solve that? It looks like I could either do `loadState.dataSource.subsources` or `loadState.subsources`
 
```ts
getMeshSource() {
    const {layer} = this;
    for (const dataSource of layer.dataSources) {
      const {loadState} = dataSource;
      if (loadState instanceof LoadedLayerDataSource) {
        const {subsources} = loadState.dataSource;
        const graphSubsource = subsources.filter(subsource => subsource.id === 'graph')[0];
        if (graphSubsource && graphSubsource.subsource.segmentationGraph) {
          if (graphSubsource.subsource.segmentationGraph !== this.graph) {
            continue;
          }
        }
        const meshSubsource = subsources.filter(subsource => subsource.id === 'mesh')[0];
        if (meshSubsource) {
          return meshSubsource.subsource.mesh;
        }
      }
    }
    return undefined;
  }
```